chat.js

```diff
@@ -389,6 +397,9 @@ class CommandContext {
 		if (room.filterCaps && user.name.match(/[A-Z\s]{6,}/)) {
 			return this.errorReply(`Your username contains too many capital letters, which this room doesn't allow.`);
 		}
+        if (room.filterEmojis && emojiRegex.exec(user.name)) {
```

Comment:
Unicode emoji definition is too crazy, I feel like. Like ©️, ®️, ‼️, ⁉️ and ™️ are considered Emojis™️.